natlas-server/app/models/rescan_task.py

```diff
@@ -18,7 +18,7 @@ class RescanTask(db.Model, DictSerializable):  # type: ignore[misc, name-defined
     date_dispatched = db.Column(db.DateTime, index=True)
     complete = db.Column(db.Boolean, default=False, index=True)
     date_completed = db.Column(db.DateTime, index=True)
-    scan_id = db.Column(db.String(256), index=True, unique=True)
```

Comment:
This is interesting because it looks like this 256 character scan id was never actually migrated, so it's just been waiting for someone to make a migration for many years. I just reverted it back to what the migrations say it should be in order to minimize drift.